.travis.yml

```diff
@@ -2,8 +2,8 @@ language: node_js
 
 node_js:
   - 'node'
+  - '12'
   - '10'
-  - '8'
```

Comment:
Just `rm .travis.yml`?

Reply:
You mean, disable Travis entirely? I thought @weswigham mentioned a reason that we should keep it around. Or maybe that was pipelines.